test/integration/testpaths/testpath_multiple_snapshot.py

```diff
@@ -0,0 +1,279 @@
+            )
+            # Create Service offering
+            cls.service_offering_zwps = ServiceOffering.create(
+                cls.apiclient,
+                cls.testdata["service_offering"],
+                tags=ZONETAG1
+            )
+
+            cls.disk_offering_zwps = DiskOffering.create(
+                cls.apiclient,
+                cls.testdata["disk_offering"],
+                tags=ZONETAG1
+            )
+
+            cls._cleanup = [
+                cls.account,
+                cls.service_offering_zwps,
+                cls.disk_offering_zwps,
+            ]
+        except Exception as e:
+            cls.tearDownClass()
+            raise e
+        return
+
+    @classmethod
+    def tearDownClass(cls):
+        try:
+            cleanup_resources(cls.apiclient, cls._cleanup)
+        except Exception as e:
+            raise Exception("Warning: Exception during cleanup : %s" % e)
+
+    def setUp(self):
+
+        self.cleanup = []
+        if self.skiptest:
+            self.skipTest("Skip test as setup is either not VMWare or  \
+                    having less than 2 clusters %s" % self.hypervisor)
+
+        self.apiclient = self.testClient.getApiClient()
+        self.dbclient = self.testClient.getDbConnection()
+        self.pools = []
+
+    def tearDown(self):
+        try:
+            for storagePool in self.pools:
+                StoragePool.update(self.apiclient, id=storagePool.id, tags="")
+
+            cleanup_resources(self.apiclient, self.cleanup)
+        except Exception as e:
+            raise Exception("Warning: Exception during cleanup : %s" % e)
+        return
+
+    @attr(tags=["advanced", "basic"], required_hardware="true")
+    def test_01_multiple_snapshot_in_zwps(self):
+        """ Test multiple volume snapshot in zwps
+
+        # 1. Verify if setup has a ZWPS and 2 CWPS
+        # 2. Deploy a VM with data disk in ZWPS
+        # 1. Verify ROOT and DATA Disk of the VM is in ZWPS.
+        # 2. Take a snapshot of VM.
+        # 3. Create Multiple Snapshots till operation fails.
+        """
+        try:
+
+            self.pools = StoragePool.list(self.apiclient, zoneid=self.zone.id)
+            if len(list(storagePool for storagePool in self.pools
+                        if storagePool.scope == "ZONE")) < 1:
+                self.skipTest("There must be at least one zone wide\
+                storage pools available in the setup")
+
+            if len(list(storagePool for storagePool in self.pools
+                        if storagePool.scope == "CLUSTER")) < 2:
+                self.skipTest("There must be at atleast two cluster wide\
+                storage pools available in the setup")
+        except Exception as e:
+            self.skipTest(e)
+
+        # Adding tags to Storage Pools
+        zone_no = 1
+        self.debug("Storage Pools: %s" % self.pools)
+        self.zone_pool = []
+        for storagePool in self.pools:
```

Comment:
No need to tag  all the zone wide pool , only one tagged pool is enough for this test case .
You can just update the 1st zone wide pool .